rag-and-text2sql-maharishi/workflow.py

```
@@ -0,0 +1,200 @@
+import os
+from typing import Dict, List, Any, Optional
+
+from llama_index.llms.ollama import Ollama
+from llama_index.core.tools import BaseTool
+from llama_index.core.llms import ChatMessage
+from llama_index.core.llms.llm import ToolSelection, LLM
+from llama_index.core.workflow import Workflow, Event, StartEvent, StopEvent, step, Context
+from llama_index.core.base.response.schema import Response
```

Ruff (0.8.2)

`llama_index.core.base.response.schema.Response` imported but unused

Remove unused import: `llama_index.core.base.response.schema.Response`

(F401)